entity-management-definitions/src/main/java/eu/europeana/entitymanagement/normalization/EntityFieldsCompleteValidator.java

```diff
@@ -68,23 +70,27 @@ public boolean isValid(Entity entity, ConstraintValidatorContext context) {
         }
         boolean returnValueLocal = true;
 
-        Class<?> fieldType = field.getType();
+        Class<?> fieldJavaType = field.getType();
```

Comment:
better use the field object itself as it provides more information about the field, this might be required especially for nested objects

Reply:
Improved according to the new changes.